src/Command/Ingester/IngesterCommand.php

```diff
@@ -73,18 +73,19 @@ protected function configure(): void
         );
 
         $this->addArgument(
-            'source',
+            'path',
             InputArgument::REQUIRED,
-            sprintf(
-                'Source type to ingest from, possible values: ["%s"]',
-                implode('", "', array_keys($this->sourceRegistry->all()))
-            )
+            'Source path to ingest from'
         );
 
         $this->addArgument(
-            'path',
-            InputArgument::REQUIRED,
-            'Source path to ingest from'
+            'source',
+            InputArgument::OPTIONAL,
+            sprintf(
```

Comment:
Just a small note, nothing to change for now: could you please include such changes into additional PR? Because they're increasing the PR size, and not really related to the ticket scope.